lib/OpenQA/Task/Job/Restart.pm

```diff
@@ -17,6 +17,7 @@ sub restart_delay { $ENV{OPENQA_JOB_RESTART_DELAY} // 5 }
 sub restart_openqa_job ($minion_job, $openqa_job) {
     my $cloned_job_or_error = $openqa_job->auto_duplicate;
     my $is_ok = ref $cloned_job_or_error || $cloned_job_or_error =~ qr/(already.*clone|direct parent)/i;
+    OpenQA::Events->singleton->emit_event('job_restart', data => {id => $openqa_job->id}) if $is_ok;
```

Comment:
This event does not contain what the event for a "regular" job restart contains.
Please run the test and look at the event data. It needs to have the old id plus the new id(s).